### PR TITLE
Drawl capitalization coldfix (part 2: rise of accidentally commiting to master)

### PR DESCRIPTION
**Pull request by UBlueberry**
## About the PR

A sort of hotfix that fixes all-caps words not being regex'd, and "And" not being regex'd while using the Southern Drawl trait.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

fix https://github.com/space-wizards/space-station-14/issues/26637

### CODE REVIEW

**Review by ficcialfaint — DISMISSED** [review]
You're still not covering messages with mixed cases (e.g. "iNg", "iNg", etc.) `Regex.Replace()` can take RegexOptions as a fourth parameter which has `RegexOptions.IgnoreCase` that as the name suggests ignores the case

**Comment by Dutch-VanDerLinde** [comment]
instead of issue number put #26637

**Comment by UBlueberry** [comment]
oh, thanks lol im a little slow

File: Content.Server/Speech/EntitySystems/SouthernAccentSystem.cs [hunk 1]
```diff
@@ -21,8 +21,15 @@ private void OnAccent(EntityUid uid, SouthernAccentComponent component, AccentGe
...
+
         message = Regex.Replace(message, @"\band\b", "an'");
+        message = Regex.Replace(message, @"\bAnd\b", "An'");
+        message = Regex.Replace(message, @"\bAND\b", "AN'");
+
         message = Regex.Replace(message, "d've", "da");
+        message = Regex.Replace(message, "D'VE", "DA");
+
         args.Message = message;
     }
```
Comment (metalgearsloth) [inline Content.Server/Speech/EntitySystems/SouthernAccentSystem.cs:34]: One of the other accents keeps casing, look at that rather than baking these in.

**Comment by AJCM-git** [comment]
This has been inactive for a while, reopen if you come back to this

**Comment by UBlueberry** [comment]
okay. sorry, smartgit's license thing expired a while ago and i gave up on coding, but i'll try and look at this again. can i still open this even while the content freeze is still freezed?

**Comment by EmoGarbage404** [comment]
sure.

**Comment by UBlueberry** [comment]
aw crap i keep pressing too many buttons

**Comment by UBlueberry** [comment]
well ill be back with a new branch and new PR i guess cause now i have to figure out how to use tortoise git

**Comment by UBlueberry** [comment]
let me try something i might've gotten it to work

**Pull request merged**